fluent-bundle/src/bundle.rs

```diff
@@ -381,6 +382,7 @@ impl<R, M: MemoizerKind> FluentBundleBase<R, M> {
     pub fn set_formatter(&mut self, func: Option<fn(&FluentValue, &M) -> Option<String>>) {
         if let Some(f) = func {
             self.formatter = Some(f);
+            println!("SET");
```

Comment:
needs to be removed? (maybe replace with a descriptive `debug!()` if you have to)